engine/src/main/java/org/pentaho/di/job/entries/deletefiles/JobEntryDeleteFiles.java

```diff
@@ -319,26 +323,29 @@ boolean processFile( String path, String wildcard, Job parentJob ) {
               BaseMessages.getString( PKG, "JobEntryDeleteFiles.TotalDeleted", String.valueOf( totalDeleted ) ) );
           }
           isDeleted = true;
-        } else {
+        }
```

Comment:
I don't think this follows our code standards; why did this change?

Reply:
Updated format according to CheckStyle standards.